src/Auth.ts

```diff
@@ -320,13 +326,18 @@ export class Auth {
 
         // convert a PKCS#8 ASN.1 PrivateKeyInfo to PEM
         cert = pki.privateKeyInfoToPem(privateKeyInfo);
+
+        let certBags: any = p12Parsed.getBags({bagType: pki.oids.certBag});
```

Comment:
Does this need to be `any`? Don't we get back type from the `getBags` method?

Reply:
I did this because of the almost the same code a couple of lines higher already existed so I just kept the coding style. They describe it as an object but it doesn't come back as it's own type. I copied the object over, looks less clean but it is not `any`.